Uc2 (#52)
materials-marketplace/user-docs: docs/source/ucs/sa_chemical_balancer_tut .md

# Standalone app: Chemical balancer

## Description

> The chamical balancer app is a Python application that allows usert to balance chemical reactions.

Author: ```suggestion
The chemical balancer app is a Python application that allows users to balance chemical reactions.
```

This app takes unbalanced chemical equations as input and generates the corresponding balanced equation as output.

> ## How ro run the app

Author: ```suggestion
## How to run the app
```

### Inputs

> When you open the app you will be presented by the following screen

Author: ```suggestion
When you open the app you will be presented with the following screen
```

![cb_app|200x100](../_static/img/ucs/standalone_apps/balancer_input.png)

On this screen, you will find the necessary fields to input the chemical equation you wish to balance. The interface provides the following sections:

- **Reactants**: Here, you can enter the chemical substances that act as the starting materials or reactants in the reaction. Simply input the relevant compounds or elements in this section.

> - **Productsr**: This section allows you to input the chemical substances that are formed as a result of the reaction. Enter the products of the reaction in this field.

Author: ```suggestion
- **Products**: This section allows you to input the chemical substances that are formed as a result of the reaction. Enter the products of the reaction in this field.
```
